rusk/src/lib/chain.rs

```diff
@@ -0,0 +1,20 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use node::database::rocksdb::Backend;
+use node::network::Kadcast;
+
+use std::sync::Arc;
+
+use crate::Rusk;
+
+#[derive(Clone)]
+pub struct RuskNode(pub node::Node<Kadcast<255>, Backend, Rusk>);
```

Comment:
This seems like just an ancillary structure for `DataSources`. Why not just define it there?

Reply:
It's not just for datasources.

It's indeed required by that, in order to cast the DB into a concrete type.
But after that, it's used by the `main` itself in order to spawn.

So I decided to define `RuskNode` in `lib/chain` and implement `RuskNode::handle_request` into the `http` module